sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorSession.java

```diff
@@ -148,6 +149,11 @@ void dispose(ErrorCondition errorCondition) {
 
         openReceiveLinks.forEach((key, link) -> link.dispose(errorCondition));
         openSendLinks.forEach((key, link) -> link.dispose(errorCondition));
+
+        if (transactionCoordinator.get() != null) {
+            transactionCoordinator.get();
+        }
```

Comment:
What is this doing? Why do we null check and just do a get without assigning it to anything or doing anything with the return value?

Reply:
left over, removing.